wherehows-web/app/templates/datasets/dataset.hbs

```diff
@@ -62,10 +62,7 @@
               class="dataset-owner-list"
               shouldShowDatasetHealth=shouldShowDatasetHealth}}
 
-            {{visualization/charts/score-gauge
-              class="dataset-health-score"
-              score=83
-              title="Health Score:"}}
+            {{datasets/containers/health-score-gauge urn=encodedUrn}}
```

Comment:
minor comment, but in my recent tasks, I transitioned to implementing my container components as contextual components. It's easier to test and definitely solves the problem of reusability, and I think it makes more sense /is more semantic in that they only yield data and show a loader when it's pending, `dataset-lineage-downstreams.hbs` is a recent example

Reply:
Interesting, def agree with the easier to test sentiment. Will think about it to see if I have any thoughts.